doc_source/sam-resource-function.md

```diff
@@ -192,10 +192,10 @@ The runtime environment\.
 *AWS CloudFormation Compatibility*: This property is passed directly to the `[Runtime](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-runtime)` property of an `AWS::Lambda::Function`\.
 
  `Tags`   <a name="sam-function-tags"></a>
-A map \(string to string\) that specifies the tags added to the Lambda function and the corresponding IAM execution role\. Keys and values are limited to alphanumeric characters\. Keys can be 1 to 127 Unicode characters in length and cannot be prefixed with aws:\. Values can be 1 to 255 Unicode characters in length\.  
-*Type*: Map  
+A map (string to string) that specifies the tags to be added to this function\. Keys and values are limited to alphanumeric characters\. Keys can be 1 to 127 Unicode characters in length and cannot be prefixed with `aws:`\. Values can be 1 to 255 Unicode characters in length\. When the stack is created, SAM will automatically add a `lambda:createdBy:SAM` tag to this Lambda function\. Tags will also be applied to default roles generated by the function\.
+*Type*: Map of `string` to `string`
 *Required*: No  
-*AWS CloudFormation Compatibility*: This property is similar to the `[Tags](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-tags)` property of an `AWS::Lambda::Function`\. The Tags property in AWS SAM consists of Key:Value pairs\. In AWS CloudFormation it consists of a list of Tag objects\. When the stack is created, AWS SAM automatically adds a `lambda:createdBy:SAM` tag to this Lambda function and the corresponding IAM execution role\.
+*AWS CloudFormation Compatibility*: This property is different from the `Tags` property of an `AWS::Lambda::Function`\. The Tags property in AWS SAM consists of Key:Value pairs ` key: value ` while in AWS CloudFormation it consists of a list of Tag objects\. When the stack is created, AWS SAM automatically adds a `lambda:createdBy:SAM` tag to this Lambda function and the corresponding IAM execution role\.
```

Comment:
The intention here is to callout the difference, while still likening both implementations of Tags